templates/prelogin/home.html

```diff
@@ -7,9 +7,19 @@
         $(function(){
             var signUpForm = $('form.form-signup');
             var signUpFormHandler = function() {
-                _kmq.push(['record', 'Clicked "Sign up"', {}, function(){
+
+                var callback = _.once(function(){
                     signUpForm.off('submit', signUpFormHandler).submit();
-                }]);
+                });
+
+                // Track event on KISSmetrics
+                _kmq.push(['record', 'Clicked "Sign up"', {}, callback]);
```

Comment:
Is it possible to make a wrapper for Kissmetrics including this callback logic (similar to GA wrapper that we have), so that it's cleaner everywhere else?